### ISSUE TITLE
Failed to load browser extensions from directory spam

### ISSUE THREAD

**Issue by bpasero** [electron/electron#15991]
* Output of `node_modules/.bin/electron --version`: 2.0.x
* Operating System (Platform and Version): macOS

**Expected Behavior**
No error is printed to the console.

**Actual behavior**
An error is printed to the console when starting Electron:

```
Failed to load browser extensions from directory: /Users/bpasero/Library/Application Support/code-oss-dev/DevTools Extensions
{ Error: ENOENT: no such file or directory, open '/Users/bpasero/Library/Application Support/code-oss-dev/DevTools Extensions'
    at Object.fs.openSync (fs.js:577:3)
    at Object.module.(anonymous function) [as openSync] (ELECTRON_ASAR.js:166:20)
    at Object.fs.readFileSync (fs.js:483:33)
    at Object.fs.readFileSync (ELECTRON_ASAR.js:563:29)
    at App.<anonymous> (/Users/bpasero/Development/monaco/.build/electron/Code - OSS.app/Contents/Resources/electron.asar/browser/chrome-extension.js:419:52)
    at Object.onceWrapper (events.js:273:13)
    at App.emit (events.js:187:15)
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path:
   '/Users/bpasero/Library/Application Support/code-oss-dev/DevTools Extensions' }
```

The code in `chrome-extension.js:419` is spamming the console when running with logging enabled. That log is not very useful if the folder does not exist in the first place which should always be an expected condition.

**Comment by bpasero** [comment]
PR: https://github.com/electron/electron/pull/15992

**Comment by zcbenz** [comment]
Closed with  #15992.
